src/toil/test/src/fileStoreTest.py

```diff
@@ -12,6 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from __future__ import absolute_import
+from __future__ import print_function
```

Comment:
Many of these old python2 imports have been removed or reordered.  You might have to do a pull from master to bring things up to date.